src/Context/Components/ThemeContext.tsx

```diff
@@ -0,0 +1,52 @@
+import { createContext, useState, ReactNode, useEffect } from "react";
+
+export interface ITheme {
+  isDarkMode: boolean;
+  setIsDarkMode: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
We don't need to expose the setter function from the context, as we are already exposing the `toggleTheme`  function